lib/pbench/test/unit/server/test_shell_cli.py

```diff
@@ -219,7 +227,10 @@ def run(args, cwd: Optional[str] = None) -> subprocess.CompletedProcess:
         ret_val = shell.main()
 
         assert ret_val == 42
-        assert not user_site or find_called[0]
+        assert not user_site or called[0] == "find_the_unicorn"
+        assert called[-3] == "wait_for_uri"
+        assert called[-3] == called[-2]
+        assert called[-1] == "init_indexing"
```

Comment:
Alternately,
```
        expected_calls = ["find_the_unicorn"] if user_site else []
        expected_calls += ["wait_for_uri", "wait_for_uri", "init_indexing"]
        assert called == expected_calls
```
In the failure case, I think this produces better context, since PyTest will report the whole lists.

Reply:
Fixed.